lib/modules/rest/index.js

```diff
@@ -325,8 +325,16 @@ Rest.prototype.handleRequest = function(req, res, $happn, $origin) {
               return;
             }
 
-            if (body.parameters[parameter.name]) args.push(body.parameters[parameter.name]);
-            else args.push(null);
+            if (body.parameters[parameter.name]) {
```

Comment:
this sequence would allow the REST request params to override the built-in behaviour that injects request details into $req_*. Is this intentional?

Reply:
Thanks for that catch - it was not intentional. I changed the sequence.